python/gtsam/specializations.h

```diff
@@ -13,3 +13,5 @@ py::bind_vector<std::vector<gtsam::BinaryMeasurement<gtsam::Unit3> > >(m_, "Bina
 py::bind_map<gtsam::IndexPairSetMap>(m_, "IndexPairSetMap");
 py::bind_vector<gtsam::IndexPairVector>(m_, "IndexPairVector");
 py::bind_map<gtsam::KeyPairDoubleMap>(m_, "KeyPairDoubleMap");
+py::bind_vector<gtsam::CameraSet<gtsam::PinholeCamera<gtsam::Cal3_S2> > >(m_, "CameraSetCal3_S2");
+py::bind_vector<gtsam::CameraSet<gtsam::PinholeCamera<gtsam::Cal3Bundler> > >(m_, "CameraSetCal3Bundler");
```

Comment:
Again, perhaps not needed since the subclassing mechanism should give you access to the `vector` methods.

Reply:
same: ask @ProfFan , this might be above Sushmita's (and my) paygrade. @ProfFan needs to be documented !!!

Reply:
This is a case that has never been encountered before, I'll document this case shortly after this week (too busy)